src/assets/scripts/client/game/GameController.js

```diff
@@ -109,6 +115,8 @@ class GameController {
      * @method enable
      */
     enable() {
+        this._eventBus.on(EVENT.SET_THEME, this._setTheme);
```

Comment:
if there isn't one already, be sure to add a `disable()` method that removes this event listener:
```javascript
disable() {
  this._eventBus.off(EVENT.SET_THEME, this._setTheme);

  return this;
}
```